tests/run-pass/panic/good_unwind.rs

```diff
@@ -0,0 +1,27 @@
+#![feature(c_unwind, unboxed_closures, unwind_attributes)]
```

Comment:
This test looks redundant with `exported_symbol_unwind_allowed.rs`.

Reply:
In fact this can probably also be merged with `exported_symbol_good_unwind.rs`, right?

Reply:
I've removed it.

> In fact this can probably also be merged with `exported_symbol_good_unwind.rs`, right?

This is... not really related to "exported symbols": they are not `#[no_mangle]`/`#[export_name = "..."]` functions.